torch/package/_digraph.py

```diff
@@ -83,3 +86,53 @@ def __contains__(self, n):
             return n in self._node
         except TypeError:
             return False
+
+    def forward_transitive_closures(self, src: str) -> Set[str]:
```

Comment:
nit: shouldn't be plural, so `forward_transitive_closure`

Reply:
done